sailthru_content/fixups.csv

```diff
@@ -1,19 +1,8 @@
-course-v1:MITx+6.00.1x+2T2016,var,course_start,2016-08-30
-course-v1:MITx+6.00.1x+2T2016,var,marketing_url,https://www.edx.org/course/introduction-computer-science-mitx-6-00-1x-8
-course-v1:MandarinX+MX801x+2T2016,var,course_start,2016-08-22
-course-v1:MandarinX+MX801x+2T2016,var,marketing_url,https://www.edx.org/course/mandarin-chinese-business-mandarinx-mx801x
-course-v1:IEEEx+MGT_MDW+3T2016,var,marketing_url,https://www.edx.org/course/managing-multigenerational-diverse-ieeex-mgtmdw-0
-course-v1:IEEEx+MGT_MDW+3T2016,var,course_start,2016-08-31
-course-v1:ASUx+MAT117x+1T2016,var,course_end,2016-12-31
-course-v1:NYIF+YCA2015.1.2x+2016,var,course_end,2016-12-31
-course-v1:NYIF+YCA2015.1.2x+2016,,expire_date,2016-12-31
 course-v1:UBCx+UseGen.1x+1T2016,var,course_end,2017-12-31
 course-v1:UBCx+UseGen.1x+1T2016,,expire_date,2017-12-31
 course-v1:AdelaideX+HumBio101x+1T2016,var,course_end,2017-12-31
 course-v1:AdelaideX+HumBio101x+1T2016,,expire_date,2017-12-31
 course-v1:UPValenciaX+BSP101x+3T2015,var,course_end,2017-12-31
 course-v1:UPValenciaX+BSP101x+3T2015,,expire_date,2017-12-31
 course-v1:BerkeleyX+CS110x+2T2016,var,course_end,2017-12-31
-course-v1:BerkeleyX+CS110x+2T2016,,expire_date,2017-12-31
-course-v1:HKUSTx+EBA101x+3T2016,var,course_start,2016-09-13
-course-v1:Microsoft+DEV218x+3T2016,var,course_start,2016-09-19
\ No newline at end of file
+course-v1:BerkeleyX+CS110x+2T2016,,expire_date,2017-12-31
```

Comment:
This is a temporary override requested by marketing on some of the missing data in the course catalog api. It's a quick dirty way to fix a people problem